packages/shared-ui/src/components/index.ts

```diff
@@ -0,0 +1,6 @@
+export {
```

Comment:
I think it's fine if we just keep all the files (`Header`, `HeaderMenu`, `ExternalLink`, `Transition`) in the `components` folder. The library probably won't grow too large (at least for now) so it should be fine.